.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 npm-debug.log
+.idea
```

Comment:
Could you remove that?

Reply:
Well, if you want, but it's just a .gitignore file, nothing to worry about !
This is from my IDE.

Reply:
Done on the last commit, I've squashed it !